Documentation/flexvolume.md

```diff
@@ -60,6 +60,12 @@ The kubelet on GKE is already configured to use that directory.
 
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
 
+### Kubespray
+Kubespray uses a non-standard FlexVolume [plugin directory](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/node/defaults/main.yml#L55): `/var/lib/kubelet/volume-plugins`.
```

Comment:
Could you expand the link to `FlexVolume` (`FlexVolume plugin directory` as the link text).